scripts/utils/trading_strategy_helpers.js

```diff
@@ -294,15 +297,15 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return buildBundledTransaction(transactions)
   }
 
-  const calculateBuyAndSellAmountsFromPrice = function(price, targetToken) {
+  const calculateBuyAndSellAmountsFromPrice = function(price, decimals) {
```

Comment:
I'd consider having two input values instead of just one for clarity: `stableTokenDecimals` and `targetTokenDecimals`.